src/set_theory/ordinal/basic.lean

```diff
@@ -1391,9 +1390,12 @@ begin
   rw lt_univ',
   split,
   { rintro ⟨β, e⟩,
-    exact ⟨#β, lift_mk_eq.{u _ (v + 1)}.2 e⟩ },
+    use #β,
+    rwa [lift_umax_eq.{u v (v + 1)}, lift_mk_eq] },
   { rintro ⟨c, hc⟩,
-    exact ⟨⟨c.out, lift_mk_eq.{u _ (v + 1)}.1 (hc.trans (congr rfl c.mk_out.symm))⟩⟩ }
+    refine ⟨⟨c.out, lift_mk_eq.1 _⟩⟩,
+    rw ←lift_umax_eq.{u v (v + 1)},
+    exact hc.trans (congr rfl c.mk_out.symm) }
```

Comment:
What made this proof get longer?

Reply:
This was the single situation in which the extra universe generality in `lift_mk_eq` actually helped. Note that even then, this new proof is only a few (31) characters longer.

Reply:
Does the proof become shorter if you replace `univ.{v (max u (v + 1))}` by `univ.{v u}`? We have `theorem univ_umax : univ.{u (max (u+1) v)} = univ.{u v}` above so I think we should choose the shortest spelling among all equivalent ones.

Reply:
Not really, but it's still nicer to write it down this way.

On more careful analysis, the proof hasn't really changed. All that's happened is that `lift_umax_eq` is no longer implicitly a part of `lift_mk_eq`.